app/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+	<body>
```

Comment:
looks like you're not using two-space indentation. In the lower-right corner of your sublime window, there's a menu icon to select spacing. Choose 2-space indentation instead of 4-space or tab. It makes it much easier to read, especially on GitHub.